_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5663,15 +5661,15 @@ mfxStatus ConfigureExecuteParams(
                             targetFourCC == MFX_FOURCC_BGR4)
                         {
                             executeParams.iBackgroundColor = ((mfxU64)0xff << 48)|
-                               ((mfxU64)VPP_RANGE_CLIP(extComp->R, 0, 255) << 32)|
-                               ((mfxU64)VPP_RANGE_CLIP(extComp->G, 0, 255) << 16)|
-                               ((mfxU64)VPP_RANGE_CLIP(extComp->B, 0, 255) <<  0);
+                               (VPP_RANGE_CLIP<mfxU64>(extComp->R, 0, 255) << 32)|
+                               (VPP_RANGE_CLIP<mfxU64>(extComp->G, 0, 255) << 16)|
+                               (VPP_RANGE_CLIP<mfxU64>(extComp->B, 0, 255) <<  0);
```

Comment:
Before it compared as U8 and casted result to U64, now it casts first then compared as u64. Result is the same but operations more consuming (unless optimized to previous by compiler).

Reply:
ok, will return to u8